public/images/Icons.js

```diff
@@ -0,0 +1,46 @@
+const React = require("react");
+const InlineSVG = require("svg-inline-react");
+const { DOM: dom } = React;
+
+const DomainIcon = props => {
```

Comment:
Any reason we need all these components now? We could just use `InlineSVG({ src: DomainIcon })` directly in the component that needs it. That would reduce the need to manage this file.